python/src/aiconfig/editor/client/src/Editor.tsx

```diff
@@ -242,6 +254,64 @@ export default function Editor() {
               justifyContent: "space-between",
               alignItems: "center",
             },
+            ".parametersContainer": {
+              maxWidth: "1250px",
+              maxHeight: "-webkit-fill-available",
+              margin: "16px auto",
+              padding: "0",
+              backgroundColor: "rgba(226,232,255,.1)",
+              borderRadius: "4px",
+              border: "1px solid rgba(226,232,255,.1) !important",
+              button: {
+                ":hover": {
+                  backgroundColor: "rgba(226,232,255,.1)",
+                },
+              },
+              input: {
+                border: "1px solid rgba(226,232,255,.1)",
+                backgroundColor: "#060c21",
+                borderRadius: "4px",
+                ":focus": {
+                  outline: "solid 1px #ff1cf7 !important",
+                  outlineOffset: "-1px",
+                },
+              },
+              textarea: {
+                border: "1px solid rgba(226,232,255,.1)",
+                backgroundColor: "#060c21",
+                borderRadius: "4px",
+                ":focus": {
+                  outline: "solid 1px #ff1cf7 !important",
+                  outlineOffset: "-1px",
+                },
+              },
+            },
+            ".addParameterButton": {
+              position: "sticky",
+              left: "0",
+              bottom: "0",
+              margin: "16px 0 0 0",
+              background: "#ff1cf7",
+              "&:hover": {
+                background: "#ff46f8",
+              },
+            },
+            ".mantine-InputWrapper-label": {
+              display: "none",
+            },
```

Comment:
these are going to apply to every mantine input, right? I'm pretty sure we don't want that -- is this intended only for the prompt name / input label?

Reply:
I moved it to be only applied to input fields within the cell container